src/components/Toggle/Toggle.module.css

```
@@ -23,16 +24,18 @@
   background-color: var(--color-primary);
 }
 
-.on:hover {
-  background-color: var(--color-primary-darken);
-}
-
 .off {
   background-color: var(--color-ubie-black-400);
 }
 
-.off:hover {
-  background-color: var(--color-ubie-black-700);
+@media (pointer: fine) {
+  .on:hover {
+    background-color: var(--color-primary-darken);
+  }
+
+  .off:hover {
+    background-color: var(--color-ubie-black-700);
+  }
```

Change this if the discussion passes
```suggestion
/* Reset on touch devices, it doesn't add specificity */
@media (hover: none) {
  .on:hover {
    background-color: var(--color-primary);
  }

  .off:hover {
    background-color: var(--color-ubie-black-400);
  }
}
```